pyecharts/charts/basic_charts/effectscatter.py

```diff
@@ -6,9 +6,10 @@
 
 class EffectScatter(RectChart):
     """
-    <<< 带有涟漪特效动画的散点图 >>>
+    <<< Scatter plots with ripple effects animation >>>
 
-    利用动画特效可以将某些想要突出的数据进行视觉突出。
+    Use animation effects to visually highlight
+    certain data that you want to highlight.

```

Comment:
Use animation effects to visually highlight 
certain data that you want to highlight. 

->

Use animation effects to visually highlight designated data set.